### ISSUE TITLE
Game crashed when the smeltery has too many kinds

### ISSUE THREAD

**Issue by dlfrnjs8878** [SlimeKnights/TinkersConstruct#3423]
I put some coal to the controller just before the crash, that's all.

there was roughly 3000 b/ 8000 b occupied,  so I don't think the error came from the smeltery overflow.

but one thing, too many kinds of metal was inside that the controller could barely indicate all the content.

after the input, 
, crash began with the controller spitting out coals 
and game crashes repeatedly after a while I restart the game... help!!!

https://gist.github.com/dlfrnjs8878/0b8170d5e8164f881bc39bfa5ef0562f

**Comment by TheQuadShot** [comment]
You can try the following, **HOWEVER there is no guarantee** that it will work, so make a backup/copy of your saved world!! (plus the devs may want you to try something) 

Then edit your <mcgamedir>/config/forge.cfg file and look for this line:
  
    B:removeErroringTileEntities=false
change it to true:
    B:removeErroringTileEntities=true
save it and restart minecraft

If it works ie your game launches but your smeltery controller will be gone along with everything that was in the smeltery, and you'll have to make a new one. Just don't forget to go back and set the config back to false.

**Comment by KnightMiner** [comment]
According to the crash, some mod is adding a broken alloy recipe that gives a null result. We could add a catch for that somewhere, but for now you are best off trying to find which mod does that so you can report the issue to them

**Comment by bonii-xx** [comment]
Probably one of those "only happens when two mods would add the fluid so cached fluids break" scenario.

**Comment by dlfrnjs8878** [comment]
Okay, I edited forge.cfg and smeltery's gone empty. 
some kinda alloys made a problem(as you guys said before), I'll check what made this happened.
Just wondering some of the metals from 'base metals' mod or 'modern metals' mod.
It'll take some time to find it out cuz I have something to do with assignments my professor gave while handling this, and I'm also not good at it too.
I'll post the issue later when I found the thing. nice job man


![2018-03-20_01 17 27](https://user-images.githubusercontent.com/37497057/37607923-9403e964-2bdc-11e8-90e8-ca7a9e5fd5f3.png)
(taken right after starting game and right-clicked the controller)
